### PR TITLE
Roshan

### PR DESCRIPTION
**Pull request by roshansa74**
Changes were made to the data field names.

### CODE REVIEW

**Review by utkriststha — CHANGES_REQUESTED** [review]

File: src/library/entities/Book.java [hunk 1]
```diff
@@ -5,91 +5,91 @@
...
-	public boolean iS_On_LoAn() {
-		return StAtE == sTaTe.ON_LOAN;
+	public boolean isOnLoan() {// Changed 'iS_On_LoAn' to 'isON_LOAN'
+		return state == state.ON_LOAN;//Changed 'StAtE' to 'state'
 	}
 
 	
-	public boolean iS_DaMaGeD() {
-		return StAtE == sTaTe.DAMAGED;
+	public boolean isDAMAGED() {//Changed 'iS_DaMaGeD' to 'isDAMAGED'
```
Comment (utkriststha) [inline src/library/entities/Book.java:56]: method name should be in camelBack, isDAMAGED cannot be used please use isDamaged.

File: src/library/entities/Book.java [hunk 2]
```diff
@@ -5,91 +5,91 @@
...
 		
 		
 	}
 
 
-	public void ReTuRn(boolean DaMaGeD) {
-		if (StAtE.equals(sTaTe.ON_LOAN)) 
-			if (DaMaGeD) 
-				StAtE = sTaTe.DAMAGED;
+	public void return(boolean DAMAGED) {//Changed 'ReTuRn' to 'return' and ' DaMaGeD' to 'DAMAGED'
```
Comment (utkriststha) [inline src/library/entities/Book.java:72]: Please change the method name from return to returnBook. return cannot be used as an identifier.

File: src/library/entities/Book.java [hunk 3]
```diff
@@ -5,91 +5,91 @@
...
-		this.StAtE = sTaTe.AVAILABLE;
+		this.author = author;// Changed 'AuThOr' to 'author'
+		this.title = title;// Changed 'tItLe ' to 'title'
+		this.callNo = callNo;//Changed 'CALLNO' to 'callNo'
+		this.id = id;//Changed 'iD' to 'id'
+		this.state = state.AVAILABLE;//Changed 'StAtE ' to 'state' and 'sTaTe.AVAILABLE' to 'state.AVAILABLE'
 	}
 	
 	public String toString() {
 		StringBuilder sb = new StringBuilder();
```
Comment (utkriststha) [inline src/library/entities/Book.java:26]: As the variable name should be meaning full, please use stringBuilder instead of sb.

**Review by utkriststha — CHANGES_REQUESTED** [review]
Few corrections need to be done.

**Comment by roshansa74** [comment]
a correction was made by adding identifiers, giving the meaningful variable name and method name were changed into camelBack order

**Pull request merged**